presidio-anonymizer/tests/operators/test_operators_factory.py

```diff
@@ -38,6 +55,30 @@ def test_given_anonymize_operators_class_then_we_get_the_correct_class():
             or operator.operator_type() == OperatorType.All
         )
 
+    resetOperatorsFactory()
+    del Encrypt2
```

Comment:
Up until now we used to utilize 'unittests' setup and teardown capabilities to rest or delete in between runs. Its more safe for extension of the test class, and the next developer won't have to know if he has to call deletion or reset methods (temporal coupling)